document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/data/accounts/AccountsManager.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.companieshouse.document.generator.accounts.data.accounts;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.companieshouse.api.model.accounts.Accounts;
+import uk.gov.companieshouse.environment.impl.EnvironmentReaderImpl;
+
+public class AccountsManager {
+
+    /** represents the Authorization header name in the request */
+    private static final String AUTHORIZATION_HEADER = "Authorization";
+
+    private static final RestTemplate restTemplate = createRestTemplate();
+
+    /**
+     * Try get transaction if exists
```

Comment:
Do you mean 'Try to get accounts if they exist' or similar?

Reply:
yes i do :)